tests/test_query_engine_dsl.py

```diff
@@ -12,6 +12,10 @@
 pytestmark = pytest.mark.integration
 
 
+def _as_datestring(date_obj):
+    return date_obj.strftime("%Y-%m-%d")
```

Comment:
Can you not use `str()`?

```
>>> str(date(1990, 8, 10))
'1990-08-10'
```

Reply:
huh. did not occur to me that would work

Reply:
@inglesp I've updated that now.  Are you OK with the additional changes I've made since you approved?

Reply:
:shipit: